backend/de.metas.business/src/main/java/de/metas/payment/paymentterm/PaymentTermService.java

```diff
@@ -82,12 +82,9 @@ private PaymentTermId getOrCreateDerivedPaymentTerm0(
 				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_AD_Org_ID, basePaymentTermRecord.getAD_Org_ID())
 				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_Discount2, basePaymentTermRecord.getDiscount2())
 				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_DiscountDays2, basePaymentTermRecord.getDiscountDays2())
-				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_AfterDelivery, basePaymentTermRecord.isAfterDelivery())
-				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_FixMonthCutoff, basePaymentTermRecord.getFixMonthCutoff())
-				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_FixMonthDay, basePaymentTermRecord.getFixMonthDay())
-				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_FixMonthOffset, basePaymentTermRecord.getFixMonthOffset())
+				.addEqualsFilter(I_C_PaymentTerm.COLUMNNAME_BaseLineType, basePaymentTermRecord.getBaseLineType())
```

Comment:
so FixMonthDay, offset won't be supported anymore?
is that known/required?

Reply:
Yes, won't be. Nobody is using it. Decide with Tobi to drop that.
And no window shows those fields.